package.json

```
@@ -1,22 +1,22 @@
 {
   "private": true,
   "name": "solc-bin",
+  "type": "module",
   "version": "0.0.0",
   "description": "Current and historical (emscripten) binaries for Solidity",
   "dependencies": {
     "semver": "^7.3.5"
   },
   "devDependencies": {
     "ethereumjs-util": "^7.1.3",
-    "ipfs-unixfs-importer": "^9.0.6",
-    "ipld": "^0.30.2",
-    "ipld-in-memory": "^8.0.0",
+    "blockstore-core": "^2.0.2",
+    "ipfs-unixfs-importer": "^11.0.1",
     "standard": "^16.0.4",
     "swarmhash": "^0.1.1"
   },
   "scripts": {
     "lint": "standard update",
-    "update": "./update --max-files-per-batch 1",
+    "update": "node ./update.mjs --max-files-per-batch 1",
```

You can remove the `./` ;)
```suggestion
    "update": "node update.mjs --max-files-per-batch 1",
```